_pages/en_US/homebrew-launcher-(pichaxx-usm).txt

```diff
@@ -0,0 +1,54 @@
+---
+title: "Homebrew Launcher (PicHaxx-unSAFE_MODE)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of using Seedminer for further exploitation uses your `movable.sed` file to gain access to the Homebrew Launcher using the PicHaxx exploit for the purposes of injecting an exploited WiFi profile.
+
+If your device already has access to the Homebrew Launcher (whether through pichaxx or another exploit), replace your current otherapp with an otherapp from [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest) located in the `otherapps_with_CfgS` folder in the unSAFE_MODE release archive. 
```

Comment:
Capitalize PicHaxx / follow whatever capitalization is used in the rest of the guide